calico/getting-started/kubernetes/managed-public-cloud/eks.mdx

```diff
@@ -70,12 +70,18 @@ Before you get started, make sure you have downloaded and configured the [necess
    eksctl create cluster --name my-calico-cluster --without-nodegroup
    ```
 
-1. Since this cluster will use {{prodname}} for networking, you must delete the `aws-node` daemon set to disable AWS VPC networking for pods.
```

Comment:
I don't understand why the new wording is more desirable - the former was more abstractly correct, whereas the latter is one particular reason among several for why we don't want the aws-node daemonset.

Reply:
Maybe "to ensure that nodes are not configured with AWS VPC networking for pods" would be better than "to disable"?